config/blocking.go

```diff
@@ -9,19 +9,18 @@ import (
 
 // BlockingConfig configuration for query blocking
 type BlockingConfig struct {
-	BlackLists        map[string][]string `yaml:"blackLists"`
-	WhiteLists        map[string][]string `yaml:"whiteLists"`
-	ClientGroupsBlock map[string][]string `yaml:"clientGroupsBlock"`
-	BlockType         string              `yaml:"blockType" default:"ZEROIP"`
-	BlockTTL          Duration            `yaml:"blockTTL" default:"6h"`
-	DownloadTimeout   Duration            `yaml:"downloadTimeout" default:"60s"`
-	DownloadAttempts  uint                `yaml:"downloadAttempts" default:"3"`
-	DownloadCooldown  Duration            `yaml:"downloadCooldown" default:"1s"`
-	RefreshPeriod     Duration            `yaml:"refreshPeriod" default:"4h"`
-	// Deprecated
-	FailStartOnListError  bool              `yaml:"failStartOnListError" default:"false"`
-	ProcessingConcurrency uint              `yaml:"processingConcurrency" default:"4"`
-	StartStrategy         StartStrategyType `yaml:"startStrategy" default:"blocking"`
+	BlackLists            map[string][]string `yaml:"blackLists"`
+	WhiteLists            map[string][]string `yaml:"whiteLists"`
+	ClientGroupsBlock     map[string][]string `yaml:"clientGroupsBlock"`
+	BlockType             string              `yaml:"blockType" default:"ZEROIP"`
+	BlockTTL              Duration            `yaml:"blockTTL" default:"6h"`
+	DownloadTimeout       Duration            `yaml:"downloadTimeout" default:"60s"`
+	DownloadAttempts      uint                `yaml:"downloadAttempts" default:"3"`
+	DownloadCooldown      Duration            `yaml:"downloadCooldown" default:"1s"`
+	RefreshPeriod         Duration            `yaml:"refreshPeriod" default:"4h"`
+	FailStartOnListError  bool                `yaml:"failStartOnListError" default:"false"` // Deprecated
```

Comment:
Was a little confused since I didn't saw the `// Deprecated` on my mobile phone at first. 😅

Do we currently have a code convention on this?

Reply:
Not sure, but I found it a bit confusing to have it inline cause to me it looks like everything below could be deprecated.  
Go also detects the EOL comment as a doc comment, so that info is still visible on symbol hover.

I could alternatively just add whitespace around the comment and field.

Reply:
I was also confused first, but it is also a valid syntax ☝️ : https://github.com/golang/go/issues/10909#issuecomment-136492606

Reply:
Ok, for fields I think the inline comment is preferable. 👍